Snakefile

```diff
@@ -363,6 +365,33 @@ if config["enable"].get("build_cutout", False):
             "scripts/build_cutout.py"
 
 
+if config["enable"].get("modify_cutout", False):
```

Comment:
I think we could call the option climate_projection_cutout or something like that?

I was considering if adding an option different from build_cutout may be necessary, but I believe so to avoid overwriting existing "original" cutouts

Reply:
Yeah, agree that it would be nice to keep the main workflow as safe as possible from interactions with these additions.

The option name revised. Have I understood you idea correctly? :)